WordScramble.js

```diff
@@ -1,14 +1,15 @@
 const words = ["apple", "plane", "orange", "tiger", "soccer", "basketball", "whale", "water", "lebron", "lion", "hippo", "bottle", "kyrie"];
-
+let random;
 function randomWord(){
     const randomIndex = Math.floor(Math.random()* words.length);
-    let random = words[randomIndex];
-    return random;
+    random = words[randomIndex];
+    
 }
 
 
 function scramble(){
-    let word = randomWord();
+    randomWord();
+    let word = random;
     let scrambledLetters;
     const letters = word.split('');
```

Comment:
You don't need to create a new variable called `word` just to copy over `random` and split it. Remember `.split()` does not mutate `random`. 

You can directly do this:
`letters = random.split('');`